src/index.js

```diff
@@ -0,0 +1,69 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+import loaderUtils from 'loader-utils';
+
+export function pitch(remainingRequest) { // eslint-disable-line no-unused-vars
+  const query = loaderUtils.getOptions(this) || {};
+
+  let chunkNameParam = '';
+
+  if (query.name) {
+    const options = {
+      context: query.context || this.options.context,
+      regExp: query.regExp,
+    };
+    const chunkName = loaderUtils.interpolateName(this, query.name, options);
+    chunkNameParam = `, ${JSON.stringify(chunkName)}`;
+  }
+
+  let result;
+  if (query.lazy) {
```

Comment:
If there will never be a sync option, because it doesn't make sense then leaving `lazy` is better yep :).